cmd/cagent/winui.go

```diff
@@ -163,7 +169,7 @@ func (ui *UI) TestSaveReload(testOnly bool) {
 		}
 	}
 
-	RunDialog(ui.MainWindow, ui.SuccessIcon, "Success", message+"Service restarted and all changes applied!", callback)
+	RunDialog(ui.MainWindow, ui.SuccessIcon, "Success", message+"Services restarted and you are all set up!", callback)
```

Comment:
Don't capitalise `Services`

Reply:
This starts from the new line like this:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/1025231/52474030-f57e4b80-2b9f-11e9-9fb9-f62c09d13de5.png">